src/Models/BlogModel.php

```diff
@@ -15,32 +18,36 @@ public function __construct($db)
         $this->db = $db;
     }
 
-    public function GetAllEntries(): bool
+    public function CreateNewEntry($blogPost): bool
     {
-        $query = $this->db->prepare("SELECT `*` FROM `blog-posts` WHERE deleted=0");
-        $query->execute();
-        $Blog = $query->fetchAll();
-        return $Blog;
+        $query = $this->db->prepare('INSERT INTO `blog-posts` (`title`, `author`, `date`, `post`)
+                                        VALUES (:title , :author, :date, :post)');
+        return $queryCheck = $query->execute($blogPost);
     }
 
-    public function CreateNewEntry($BlogPost): bool
+    public function ReadAllEntries(): array
     {
-        $query = $this->db->prepare("INSERT INTO `blog-posts` (`title`, `author`, `date`, `post`) VALUE (:title, :author, :date, :post)");
-        $AddNewEntry = $query->execute($BlogPost);
-        return $AddNewEntry;
+        $query = $this->db->query('SELECT `title`, `author`, `date`, `post`, `GUID`
+                                    FROM `blog-posts` 
+                                    WHERE `deleted` = 0;');
+         return $query = $query->fetchAll();

```

Comment:
you do not need to do the $query = bit, it is pointless, just return the fetch result